algorithms/merging/merge.py

```diff
@@ -278,3 +278,42 @@ def truncate(merged_intensities):
     logger.info("Total number of rejected intensities %s", n_removed)
     logger.debug(out.getvalue())
     return amplitudes, anom_amplitudes
+
+
+def delta_F_mean_over_sig_delta_F_mean_stats(anomalous_amplitudes, n_bins=20):
+    """Calculate the statistic for resolution bins and overall."""
+    vals = flex.double()
+    # First calculate dF/s(dF) per resolution bin
+    anomalous_amplitudes.setup_binner(n_bins=n_bins)
+    for i_bin in anomalous_amplitudes.binner().range_used():
+        sel = anomalous_amplitudes.binner().selection(i_bin)
+        arr = anomalous_amplitudes.select(sel)
+        vals.append(delta_F_mean_over_sig_delta_F_mean(arr))
+    # Finally calculate the overall value
+    vals.append(delta_F_mean_over_sig_delta_F_mean(anomalous_amplitudes))
+    return vals
+
+
+def delta_F_mean_over_sig_delta_F_mean(anomalous_amplitudes):
+    """Calculate < |F(+) - F(-)|> / <sigma(F(+) - F(-))> i.e. DANO/SIGDANO."""
+    diff = anomalous_amplitudes.anomalous_differences()
+    if not diff.data() or not diff.sigmas():
+        return 0.0
+    return flex.mean(flex.abs(diff.data())) / flex.mean(diff.sigmas())
```

Comment:
You could make this a single function, in line with various methods in `miller.array`, e.g.:
https://github.com/cctbx/cctbx_project/blob/5ae04d2955fa897b34ed4ec5bd8aea427aeeeafc/cctbx/miller/__init__.py#L3045-L3075

If you're feeling brave, it might be useful to add this to cctbx so that it can be used elsewhere (e.g. `xia2.merging_statistics`), and this might avoid the gymnastics of adding it to the merging stats output.